### PR TITLE
style: add owner column to drip-lists-all table (closes #974)

### LINKED ISSUE #974: Drip List names shouldn't be truncated on listing page

### PR DESCRIPTION
**Pull request by evvvritt**
closes #974 
Screenshot 2024-02-29 at 15 53 41
Screenshot 2024-02-29 at 15 53 56

### CODE REVIEW

**Comment by brandonhaslegs** [comment]
Spacing to the left of the Drip List name looks a bit too small

**Comment by evvvritt** [comment]
K, bumped it but global change based on how our table component works (but yeah think it looks better globally, and closer to the figma)

Screenshot 2024-02-29 at 16 47 06
Screenshot 2024-02-29 at 16 46 57
Screenshot 2024-02-29 at 16 47 17

**Comment by brandonhaslegs** [comment]
yes looks better!

File: src/lib/components/table/cells/drip-list-badge.cell.svelte [hunk 1]
```diff
@@ -11,6 +11,6 @@
   }
 </script>
 
-<div style="max-width: max(280px, 40vw)">
-  <DripListBadge {dripList} />
+<div style="max-width: max(240px, 28vw)">
```
Comment (efstajas) [inline src/lib/components/table/cells/drip-list-badge.cell.svelte:14]:
```suggestion
<div style:max-width="max(240px, 28vw)">
```

**Review by efstajas — APPROVED** [review]

**Pull request merged**